duckduckgo/duckduckgo-privacy-extension
# Global Privacy Control

**Reviewer:** @kdzwinel 

**CC:** @MariagraziaAlastra 

## Description:
This PR adds the ability for users to set the proposed Global Privacy Control standard, which allows users to signal to websites their preference to:
1. Not sell your personal data
2. Limit sharing of your personal data to other companies.

## Steps to test this PR:
1. Install extension
2. Visit [test site](https://global-privacy-control.glitch.me/)
3. Header should not be present. DOM signal should be present and have value of `false`
4. Go to extension settings, turn on GPC
5. Reload test site
6. Header should be present and have value of `"1"`. DOM signal should be present and have value of `true`
7. Visit site with `script-src` in their CSP like https://twitter.com. Check that DOM signal is set.
8. Repeat steps for Firefox and Chrome.


## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
